canonical/grafana-agent-k8s-operator
# Self monitoring

## Issue


This PR fixes #48

## Solution


In this PR we added:

- Prometheus alert rules
- Grafana dashboard
- New itests


Added dashboard
![imagen](https://user-images.githubusercontent.com/939888/179522838-5839f8c2-cd97-4710-ade8-0ceba9dda52b.png)




## Testing Instructions


Run itest or:

- Deploy Grafana
- Deploy Grafana agent
- Deploy Prometheus
- Relate grafana agent to prometheus
- Relate grafana agent to grafana
- Open grafana and check for Grafana agent dashboard
- Open prometheus and check for prometheus alert rules
